src/utils/vacation-type-utils.ts

```diff
@@ -0,0 +1,27 @@
+import { VacationType } from "../generated/client";
+
+/**
+ * Set the string to corresponding enum value
+ *
+ * @param typeString filter scope as string
+ */
+const getVacationTypeByString = (typeString: string) => {
+  switch (typeString) {
+    case "VACATION":
+      return VacationType.VACATION;
+    case "UNPAID_TIME_OFF":
+      return VacationType.UNPAID_TIME_OFF;
+    case "SICKNESS":
+      return VacationType.SICKNESS;
+    case "PERSONAL_DAYS":
+      return VacationType.PERSONAL_DAYS;
+    case "MATERNITY_PATERNITY":
+      return VacationType.MATERNITY_PATERNITY;
+    case "CHILD_SICKNESS":
```

Comment:
It looks like it could be worth creating an enum for these values (the cases?). In fact it looks like the the typeString is identical to the vacatioonType enum- could this be redundant?

Reply:
With this util I can avoid type assertion on setting data from select input, unless there is a better solution to it.